Specs/addDefaultMatchers.js

```diff
@@ -465,9 +465,10 @@ define([
         };
     }
 
-    function pickPrimitiveEquals(actual, expected) {
+    function pickPrimitiveEquals(actual, expected, x, y, pickWidth, pickHeight) {
         var scene = actual;
-        var result = scene.pick(new Cartesian2(0, 0));
+        var windowPosition = new Cartesian2(defaultValue(x,0), defaultValue(y,0));
```

Comment:
`Cartesian2` defaults to `0.0` by default when arguments are undefiend, so just `new Cartesian2(x, y)` should work.